### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by sanmai**
Let's try this.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -88,9 +88,12 @@ jobs:
       - run:
           name: Install infection
           command: composer require infection/infection:dev-master # Currently only master has https://github.com/infection/infection/pull/1106 merged.
+      - run:
+          name: Remove extra large coverage reports
+          command: find build/logs/coverage-xml -type f -size +100M -print -delete
       - run:
           name: Mutation coverage testing
-          command: php -d memory_limit=16G vendor/bin/infection --coverage=build/phpunit --no-progress --only-covered --threads=2
+          command: php -d memory_limit=4G vendor/bin/infection --coverage=build/logs --no-progress --only-covered --threads=2
```
Comment (sanmai) [inline .circleci/config.yml]: Psalm's own `phpunit.xml` uses `build/logs` so if I may suggest to stick with this standard of sorts.

**Comment by sanmai** [comment]
Believe or not, it passed through:
```
4450 mutations were generated:
    1071 mutants were killed
       0 mutants were not covered by tests
      66 covered mutants were not detected
       6 errors were encountered
    3307 time outs were encountered

Metrics:
         Mutation Score Indicator (MSI): 98%
         Mutation Code Coverage: 100%
         Covered Code MSI: 98%

Please note that some mutants will inevitably be harmless (i.e. false positives).

Time: 10m 56s. Memory: 1.21GB
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -88,9 +88,12 @@ jobs:
       - run:
           name: Install infection
           command: composer require infection/infection:dev-master # Currently only master has https://github.com/infection/infection/pull/1106 merged.
+      - run:
+          name: Remove extra large coverage reports
+          command: find build/logs/coverage-xml -type f -size +100M -print -delete
```
Comment (sanmai) [inline .circleci/config.yml]:
```suggestion
```
It won't work this way. You'll have to update the index, unfortunately.

**Comment by bdsl** [comment]
Closing as this PR was AFAIK just intended as a quick experiment. This repo is just my personal development fork of `vimeo/psalm`.

**Pull request closed without merging**